controller/auth.js

```diff
@@ -8,7 +8,15 @@ const { statusCode } = require("../utils");
 
 async function login(req, res) {
   if (req.user)
```

Comment:
Chỗ này chị chưa thấy compare password vậy e? Mà request sao lại nhận vào user vậy. Phải nhận email với password chứ hả?

Reply:
https://www.npmjs.com/package/bcrypt#why-is-async-mode-recommended-over-sync-mode 


<img width="788" alt="Screenshot 2020-11-28 at 14 32 25" src="https://user-images.githubusercontent.com/17758870/100517963-8a43a600-3186-11eb-8de5-acb4228f514a.png">